### PR TITLE
sdk → cli

### PR DESCRIPTION
**Pull request by janisz**
Signed-off-by: Tomasz Janiszewski <tomek@redhat.com>

### CODE REVIEW

**Comment by janisz** [comment]
* **#199** <a href="https://app.graphite.dev/github/pr/stackrox/rox-ci-image/199?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#200** <a href="https://app.graphite.dev/github/pr/stackrox/rox-ci-image/200?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @janisz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by tommartensen — APPROVED** [review]

**Comment by RTann** [comment]
> I tried to update postgres for scanner, and ran into kubectl issues due to google-cloud-sdk. Does this resolve that issue?

looks like it does the trick, thanks!

**Review by RTann — APPROVED** [review]

File: images/stackrox-test.Dockerfile
Comment (janisz) [inline images/stackrox-test.Dockerfile:31]:
```suggestion
    dnf install -y \
```

**Comment by RTann** [comment]
@janisz do you plan on merging this soon? I want to update Scanner DB to postgres15, but I want to wait until this PR is merged first https://github.com/stackrox/rox-ci-image/pull/198

**Comment by janisz** [comment]
```
docker run -it quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.63 gcloud version
Google Cloud SDK 455.0.0
alpha 2023.11.10
beta 2023.11.10
bq 2.0.98
bundled-python3-unix 3.11.6
core 2023.11.10
gcloud-crc32c 1.0.0
gke-gcloud-auth-plugin 0.5.6
gsutil 5.27
```

```
docker run -it quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.63-2-g491b4eec86 gcloud version 
Google Cloud SDK 468.0.0
alpha 2024.03.08
beta 2024.03.08
bq 2.0.101
bundled-python3-unix 3.11.8
core 2024.03.08
gcloud-crc32c 1.0.0
gke-gcloud-auth-plugin 0.5.8
gsutil 5.27
kubectl 1.26.13
```

**Pull request merged**
